build/requirements/requirements.txt

```
@@ -1,22 +1,20 @@
-flask~=1.1.1
-Flask-Login~=0.4.0
+flask~=2.1.2
+Flask-Login~=0.6.1
 Babel~=2.9.0
 Flask-Babel~=2.0.0
-Flask-Script~=2.0.3
 SQLAlchemy~=1.3.7
 Flask-SQLAlchemy~=2.3.2
 Markdown~=3.3.4
 requests~=2.21.0
 Flask-FlatPages~=0.7.2
-pygal~=2.4.0
+pygal~=3.0.0
 GitPython~=2.1.7
-Flask-WTF~=0.14.2
+Flask-WTF~=0.15.1
 PyMySQL~=0.7.11
 blinker~=1.4.0
 factory-boy~=2.9.2
 psycopg2~=2.8.4
 Flask-QRcode~=2.0.2
-typing-extensions~=3.7.2
 email_validator~=1.1.1
-Werkzeug~=1.0.0
+Werkzeug==2.0.3
```

Please use a [compatible release clause](https://peps.python.org/pep-0440/#version-specifiers) here:
```suggestion
Werkzeug~=2.0.3
```

---

Fixed in 34d26198eae2f27083038247e9fea5b6a413ba0